pages.it/common/flac.md

```diff
@@ -0,0 +1,20 @@
+# flac
+
+> Codifica, decodifica e controlla file flac.
+> Maggiori informazioni: <https://xiph.org/flac>.
+
+- Converte un file wav in un file flac(questo creerà un file flac nella medesima posizione del file wav):
+
+`flac {{percorso/al/file.wav}}`
+
+- Codifica un file wav in flac, specificando il nome del risultato:
+
+`flac -o {{percorso/all/output.flac}} {{percorso/al/file.wav}}`
```

Comment:
For this and the next example, consider using your proposal "file_originale" and "file_modificato" (maybe, "file_convertito", in this case)